src/test/java/org/openwebnet4j/test/MessageTest.java

```diff
@@ -237,4 +242,43 @@ public void testGatewayMgmt() {
         assertNull(gwMsg.getWhere());
         assertNull(gwMsg.getWhat());
     }
+
+    @Test
+    public void testEnergyManagerUnit() {
+        EnergyManagement energyMsg;
+        try {
+            energyMsg = (EnergyManagement) BaseOpenMessage.parse("*#18*51*113##");
+            assertNotNull(energyMsg);
+            assertEquals(Who.ENERGY_MANAGEMENT, energyMsg.getWho());
+            assertFalse(energyMsg.isCommand());
+            assertEquals("51", energyMsg.getWhere().value());
+            assertEquals(EnergyManagement.DIM.ACTIVE_POWER, energyMsg.getDim());
+            assertNotNull(energyMsg.getDimValues());            
+        } catch (FrameException e) {
+            System.out.println(e.getMessage());
+
+            Assertions.fail();
+        }
+    }
+
+    // @Test
```

Comment:
i suggest to remove commented code, MessageTest is intended for unit tests, this looks more as an integration test (it involves communication with real gw)

Reply:
removed